config/rds/config.go

```diff
@@ -136,6 +136,8 @@ func Configure(p *config.Provider) {
 			"master DB user. If you set autoGeneratePassword to true, the Secret" +
 			" referenced here will be created or updated with generated password" +
 			" if it does not already contain one."
+		r.MetaResource.ArgumentDocs["engine"] = "- (Required unless a `snapshotIdentifier` or `replicateSourceDb` is provided) The database engine to use. For supported values, see the Engine parameter in [API action CreateDBInstance](https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_CreateDBInstance.html). Note that for Amazon Aurora instances the engine must match the [DB Cluster](https://marketplace.upbound.io/providers/upbound/provider-aws/latest/resources/rds.aws.upbound.io/Cluster/v1beta1)'s engine'. For information on the difference between the available Aurora MySQL engines see Comparison in the [Amazon RDS Release Notes](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraMySQLReleaseNotes/Welcome.html)."
```

Comment:
@ulucinar @sergenyalcin I noticed that  `- (Required unless a `snapshotIdentifier` or `replicateSourceDb` is provided)` is omitted below in CRD specs https://github.com/upbound/provider-aws/pull/702/files#diff-ef705738ded9a9f6afaf5f1c2a524e8011b77a14e4ed036557ed7599f7eedd20R265. Can't easily find what is filtering it, any idea?

Reply:
@ytsarev I tracked down the issue you're having and noticed that dashes and optional/required information trimmed out intentionally: https://github.com/upbound/upjet/pull/93/files#diff-045ed75f3dc44f56727f149f5aee828ad3063b5ea5c0f143c7430437dec2a82dR172-R184

I don't have further context on why, so, @sergenyalcin should better answer.